solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -151,36 +146,7 @@ const getPartsLastUpdated = async () => {
 
 const { policyDocList, getDocList, clearDocList } = useSearchResults();
 
-// use "reactive" method to make urlParams reactive when provided/injected
-// selectedParams.paramString is used as the reactive prop
-const selectedParams = reactive({
-    paramString: "",
-    paramsArray: [],
-});
-
```

Comment:
The Combined Search page (this component) was created using the existing Subjects page as a starting point.  Some unnecessary and confusing logic was kept in the Search page when it was not needed; this `selectedParams` logic included.  This logic is used on the Subjects page to create the Subject Name chips that are seen in the top left corner of the Subjects page.  No such chips exist on the Search page, so this logic has been removed, simplifying the logic related to fetching subjects and documents.